javascript/javascript3/week1/GiphyApi/main.js

```diff
@@ -0,0 +1,53 @@
+const apiKey = 'TnGhS9Vu9jrTmAOpBbNXscaRtP8opJNw'
+const giphyName = document.getElementById('giphy-name');
+const btnsearch = document.getElementById('btn-giphy');
+const giphyNum = document.getElementById('giphy-num');
+const displyGiphy = document.getElementById('giphy-result')
+let giphyNameSearch = giphyName.value;
+let disGiphy = giphyNum.value
+giphyNameSearch = "hello";
+disGiphy = 25
+if (giphyNameSearch === "hello") {
```

Comment:
This if is always true looking at the code. so you do not need it

Reply:
I have removed if condition and updated.